### ISSUE TITLE
Unintended split of array with special characters

### ISSUE THREAD

**Issue by Sidewinder53** [adolfintel/speedtest#146]
When `getIP.php` returns ISPinfo with special characters (e.g. Ampersand) the `HTML_ESCAPE_MAP` in `escapeHtml` returns the corresponding HTML entity (in this case `&amp;`).

This results in an unintended split at the end of the special character, caused by function `startStop` in some example pages. The function is splitting data by semicolons: `var data=e.data.split(';');`

The data from data[4] (ISP info) leaks into data[5] (Jitter in ms):
```
data: Array(11)
0: "2"
1: ""
2: ""
3: "17.68"
4: "<IPv6 client address> - AS8881 1&amp"
5: "1 Versatel Deutschland GmbH, DE (280 km)"
6: "0.49"
7: "0"
8: "0"
9: "0.2"
10: "noID"
```

This is the result:
![image](https://user-images.githubusercontent.com/19623985/43774029-fd2cf424-9a47-11e8-94cb-6c76e2c92c45.png)
![image](https://user-images.githubusercontent.com/19623985/43774048-1412c678-9a48-11e8-8c53-87aa3c7c91a5.png)

**Comment by adolfintel** [comment]
I fixed it this morning, have you updated to the latest master?

**Comment by Sidewinder53** [comment]
I downloaded v4.6 just 40 minutes before you released v4.6u1.. 😢
Anyway, it's fixed with v4.6u1, thanks for your work.

**Comment by adolfintel** [comment]
Yea, it was a very short lived bug, I spotted it almost immediately after release.
Next update I'll have the worker output a JSON string so this will never happen again
